src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -202,4 +202,22 @@ public function count()
     {
         return \count($this->parameters);
     }
+
+    /**
+     * Returns the parameter value converted to array.
+     */
+    public function getJsonArray(string $key, ?array $default = []): array
+    {
+        $value = $this->get($key, $default);
+
+        if (\is_string($value)) {
+            $value = json_decode($value, true);
```

Comment:
What if `json_decode()` fails? What if it does not produce an array?

Reply:
I think what I need is to return the default value, if 'json_decode' wont return an array, what do you think?